azure/appservice-windows/main.tf

```diff
@@ -11,6 +11,28 @@ resource "azurerm_app_service_plan" "app" {
   }
 }
 
+resource "azurerm_virtual_network" "app" {
+  name                = "vnet-${var.project}-${var.environment}"
+  location            = local.location
+  resource_group_name = var.resource_group
+  address_space       =  ["172.22.0.0/16"]
+}
```

Comment:
```suggestion
data "azurerm_virtual_network" "vnet" {
  name                = var.vnet.name
  resource_group_name = var.vnet.resource_group_name
}
```

Reply:
A new vnet will be created on each call to this module, and that is not the goal.